commands.go

```diff
@@ -56,10 +60,18 @@ func runAgent() {
 		PfApiPath,
 	)
 
-	// Self Register
-	ok, _ := pfclient.Register(hostname, ipaddress)
-	if !ok {
-		panic(errors.New("Cannot register to pathfinder server, please check your configuration."))
+	for {
+		log.WithFields(log.Fields{}).Warn("Trying to register to pathfinder server...")
+		ok, _ := pfclient.Register(hostname,ipaddress)
+		
+		if !ok {
+			log.Error("Cannot register to pathfinder server, please check your configuration")
```

Comment:
can we add `Retrying in x seconds...`
and calculate the delay before this line

Reply:
Added the log message

Reply:
Done f46ae53